src/xtopcom/xtxpool_service_v2/src/xtxpool_service.cpp

```diff
@@ -244,14 +244,14 @@ void xtxpool_service::on_message_receipt(vnetwork::xvnode_address_t const & send
 
     XMETRICS_TIME_RECORD("txpool_network_message_dispatch");
 
-    if (message.id() == xtxpool_msg_send_receipt || message.id() == xtxpool_msg_recv_receipt || (message.id() == xtxpool_msg_push_receipt)) {
+    if (message.id() == xtxpool_v2::xtxpool_msg_send_receipt || message.id() == xtxpool_v2::xtxpool_msg_recv_receipt || (message.id() == xtxpool_v2::xtxpool_msg_push_receipt)) {

```

Comment:
txpool service has same namespace with txpool

Reply:
txpool and txpool service use different namespace currently，not modify this time.